src/vos/vos_common.c

```diff
@@ -759,6 +759,12 @@ vos_metrics_alloc(const char *path, int tgt_id)
 	if (rc)
 		D_WARN("Failed to create 'merged_size' telemetry : "DF_RC"\n", DP_RC(rc));
 
+	/* VOS aggregation failed */
+	rc = d_tm_add_metric(&vam->vam_fail_count, D_TM_COUNTER, "aggregation failures", NULL,
+			     "%s/%s/fail_count/tgt_%u", path, VOS_AGG_DIR, tgt_id);
+	if (rc)
+		D_WARN("Failed to create 'fail_count' telemetry : "DF_RC"\n", DP_RC(rc));
```

Comment:
FYI - We've updated best-practice here to use the following syntax for improved clarity.   ```suggestion
		DL_WARN(rc, "Failed to create 'fail_count' telemetry");
```

Reply:
Will there be a mass-update of all of the `D_WARN` lines?

Reply:
There could be in future but the current focus is in changing developer practices.